assets/components/src/action-card/index.js

```diff
@@ -1,9 +1,9 @@
 /**
```

Comment:
In IE11, Action Card description text overflows the container and does not break to a second line:

<img width="1595" alt="Screen Shot 2019-12-11 at 11 02 36 PM" src="https://user-images.githubusercontent.com/1477002/70681774-8e6f6e80-1c6a-11ea-8566-354e5831beab.png">

<img width="1595" alt="Screen Shot 2019-12-11 at 11 02 05 PM" src="https://user-images.githubusercontent.com/1477002/70681777-93ccb900-1c6a-11ea-8494-4fccef4e3853.png">

Reply:
Fixed in 2ab9cb5